### DISCUSSION TITLE
Welcome to Flipper-Zero Discussions!

### DISCUSSION THREAD

**Discussion by KosteQ314** [KosteQ314/Flipper-Zero#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Shrap19** [comment]
Having issues installing the Marauder firmware on my FZ Wifi ESP32 with Mac OS (Macbook Pro Ventura 13.0.1) Any help is greatly appreciated.
